GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/RepairTests.cs

```diff
@@ -49,6 +49,42 @@ public void FixesCorruptHeadSymRef()
             this.RepairWithConfirmShouldFix();
         }
 
+        [TestCase]
+        public void FixesCorruptHeadsSha()

```

Comment:
Not sure if we have any but will you add/make sure there are some tests where the ref is good and doesn't need fixed and where the reflog file for the ref is missing.